web/packages/teleport/src/Welcome/Questionnaire/Resources.tsx

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright 2023 Gravitational, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Flex, Text } from 'design';
+
+import React from 'react';
+
+import { InputLabel } from 'design/Input/InputLabel';
+
+import Image from 'design/Image';
+
+import { useTheme } from 'styled-components';
+
+import { CheckboxInput } from 'design/Checkbox';
+
+import {
+  ResourcesProps,
+  ResourceType,
+} from 'teleport/Welcome/Questionnaire/types';
```

Comment:
Nit: when importing something from the same "module" I would use relative import: 
 ```typescript
import {
   ResourcesProps,
   ResourceType,
 } from './types';
```